src/sdk/db_sdk.h

```diff
@@ -67,7 +67,7 @@ class DBSDK {
 
         std::string endpoint, real_endpoint;
         if (!GetNsAddress(&endpoint, &real_endpoint)) {
-            LOG(DFATAL) << "fail to get ns address";
+            DLOG(ERROR) << "fail to get ns address";
```

Comment:
After the PR, the `tablet server` may try to connect with the `nameserver` when `nameserver` is not alive yet (`SQLClusterRouter`). So we log the error rather than `abort` the program.